packages/decdk/test/fixture/tsconfig.json

```
@@ -7,7 +7,7 @@
     "inlineSourceMap": true,
     "inlineSources": true,
     "lib": [
-      "es2016"
+      "es2018"
```

I kinda prefer having it as:

```suggestion
      "ES2018"
```

But it changes absolutely nothing :)

---

Also - this file has a "generated by jsii" footer. If that is no longer accurate, it should be removed. If that *is* accurate, then the file should be removed & ignored.

---

I don't remember changing this file, so it must have been generated by jsii.